frontend/src/Components/NavigationWrapper/NavigationListItem.tsx

```diff
@@ -24,7 +24,13 @@ const NavigationListItem = ({
 
   return (
     <ListItem button={true} onClick={onClick} selected={itemSelected}>
-      <ListItemIcon>{children}</ListItemIcon>
+      <ListItemIcon>
+        {Array.isArray(children) ? (
```

Comment:
Gal geriau taip nedarom xD

Reply:
:( kodėl

Reply:
Nes tai yra bandymas fittinti typescript su kazkokias workaroundais :D 
Typescriptas turetu buti irankis mums, o ne policininkas :D 
Jeigu react siulo patogu API `children` tai mes ir norime naudot `children` o ne kurti warpperius kurie pataikauja typescriptui :D

Reply:
vistiek mane troublino šitas, bet jei pažėsim to `ListItemIcon` docs tai jis by design tik vieną child elementą priėma (i guess typescriptas mum tą sakė nuo pat pradžių)..
https://material-ui.com/api/list-item-icon/

Reply:
Tai tame problema buvo 🤣🤣